tests/test_ir.py

```diff
@@ -0,0 +1,26 @@
+from xdsl.ir import Block, Region
+from xdsl.dialects.arith import Addi, Subi, Constant
+from xdsl.dialects.builtin import i32
+
+
+def test_block_build():
```

Comment:
I would describe this test more like `test_ops_accessor` or something. Even though you are building a block, you are not actually doing assertions over just the built block but over regions and results, right?

Reply:
ok